app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -199,8 +200,26 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     submit,
     values,
   } = props
+
   const { arbitrator, category, collateral, funding, loadedQuestionId, outcomes, question, resolution, spread } = values
 
+  const { markets } = useGraphMarketsFromQuestion(loadedQuestionId || '')
+
+  const [currentToken, setCurrentToken] = useState({ exists: false, token: '', marketAddress: '' })
+  useEffect(() => {
+    const rinkebyDai = '0x5592ec0cfb4dbc12d3ab100b257153436a1f0fea'
+    const mainnetDai = '0x6b175474e89094c44da98b954eedeac495271d0f'
+    const array = markets.find(
```

Comment:
Probably better to use `markets.include` as it will return a boolean value, and I would change the name to something like `tokenExists` for better clarity.

Reply:
@KadenZipfel problem is that i need values from the object not just if they exist so that i can send them link to the specific market and also to register the specific market